Исправление краша при открытии комнаты с большим количеством участников

Popalay opened the pull request:
- Исправление TransactionTooLargeException передачей информации о канали через статическую переменную
- Испраление утечки броадкаст ресивера очишением RtcEngine при разрушении VoiceService

grishka commented:
А не лучше будет объект канала просто в статик класть? Через файл и сериализацию гнать не совсем понятно зачем 🤔

Popalay commented:
Можно и через статик конечно. Но в случае с префами можно пережить уничтожение процесса а просадки в чтении с таким количеством данных особо и не заметно

grishka commented:
Сделай лучше через статик, тогда смержу. Процесс всё равно не может прибиться между startService и самим запуском сервиса.

Popalay committed:
- Fix leaked broadcast receiver
- Fix TransactionTooLargeException by passing a channel info through static variable

grishka commented:
Спасибо!
